test/stripe/bank_account_test.rb

```diff
@@ -31,6 +31,12 @@ class BankAccountTest < Test::Unit::TestCase
         bank_account = bank_account.verify(amounts: [1, 2])
         assert bank_account.is_a?(Stripe::BankAccount)
       end
+
+      should "verify the account using the static method" do
+        bank_account = Stripe::BankAccount.verify("cus_123",
```

Comment:
Can we add a line here to verify that this is hitting the right url as well, like so? https://git.corp.stripe.com/stripe-private-oss-forks/stripe-ruby/blob/7c9492b3/test/stripe/account_external_accounts_operations_test.rb#L18

Reply:
Oh nice! I wanted to add something like that when I first started on this entire task - but later resigned to copy the test from the other verify method

Reply:
Done with https://github.com/stripe/stripe-ruby/pull/1418/commits/e58bb5a4832eacb99498000f2e9c9c0ec347d2a9